drivers/sensor/lsm303dlhc_magn/lsm303dlhc_magn.h

```diff
@@ -37,19 +37,26 @@
 					LSM303DLHC_MAGN_ODR_SHIFT)
 
 #if	(CONFIG_LSM303DLHC_MAGN_RANGE == 1)
-	#define LSM303DLHC_MAGN_LSB_GAUSS	1100
+	#define LSM303DLHC_MAGN_LSB_GAUSS_XY	1100
+	#define LSM303DLHC_MAGN_LSB_GAUSS_Z		980
```

Comment:
Nit: indentation

Reply:
@microbuilder - can you fix this one last nit?

Reply:
@cfriedt This is not blocking, I approved the change anyway. This is left to @microbuilder appreciation.

Reply:
Pushed a fix out.